File: 1742-widest-vertical-area-between-two-points-containing-no-points/widest-vertical-area-between-two-points-containing-no-points.py
class Solution:
    def maxWidthOfVerticalArea(self, points: List[List[int]]) -> int:


        points.sort()
        n = len(points)
        print(points)
        ans  = -1
        for i in range(1,n):
            ans = max(ans, points[i][0]- points[i-1][0])
        return ans

File: 1742-widest-vertical-area-between-two-points-containing-no-points/widest-vertical-area-between-two-points-containing-no-points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        

        Arrays.sort(points,(a,b)->{

                return a[0]- b[0];
        });
        int  n  = points.length;
        int ans = -1 ;
        for(int i=1 ;i<n ;i++){
            ans = Math.max(ans, points[i][0]- points[i-1][0]); 
        }
        return ans;
    }
}